input/drivers/switch_input.c

```diff
@@ -14,9 +14,13 @@
 
 #define MULTITOUCH_LIMIT 4 /* supports up to this many fingers at once */
 #define TOUCH_AXIS_MAX 0x7fff /* abstraction of pointer coords */
+#define SWITCH_NUM_SCANCODES 114
+
+int switch_keyboard_state[SWITCH_NUM_SCANCODES] = {};
```

Comment:
Move that array to `switch_input_t` for consistency

Reply:
fixed: changed int array `switch_keyboard_state[SWITCH_NUM_SCANCODES]` to a bool array inside switch_input_t. Renamed to `keyboard_state` for consistency.